blackbox/docs/administration/hba.txt

```diff
@@ -0,0 +1,78 @@
+.. _administration_hba:
+
+=========================
+Host Based Authentication
+=========================
+
+.. NOTE::
+
+   Host based Authentication is an Enterprise Edition feature.
+   This feature only concerns connections over ``PostgreSQL Wire Protocol``.
+
+This topic explains how to configure client connections and authentication for CrateDB.
+
+When CrateDB is started, the cluster contains one predefined ``superuser``. This user
+is referred to as ``crate``.
+
+By default, CrateDB’s configuration does not contain any host based authentication
```

Comment:
By default the optional setting `auth.host_based` is absent and therefore host based authentication is disabled. The CrateDB cluster allows any unauthenticated connections.